project_setup/branch-protection.md

```diff
@@ -22,18 +22,19 @@ In `Settings`, go to the `Branches` entry and create a rule with the following:
     - [X] Require review from Code Owners
     - [X] Restrict who can dismiss pull request reviews
   - [X] Require status checks to pass before merging
-  - [X] Require branches to be up to date before merging
-    - There may be a list of status checks under this option. We require
-    passing status checks to merge, so all status checks should generally be
-    checked as required.
-    - Please note that the list of status checks will not fully populate in a
-    new repository until the first pull request (PR) has been created.
+    - [X] Require branches to be up to date before merging
+      - There may be a list of status checks under this option. We require
+      passing status checks to merge, so all status checks should generally be
+      checked as required.
+      - Please note that the list of status checks will not fully populate in a
+      new repository until the first pull request (PR) has been created.
```

Comment:
We should probably make a note to update this blurb in an issue. The branch protection page no longer displays a selectable list of recent tests and instead you must search.

Reply:
See also #58, which i just created.

Reply:
@mcdonnnj - Please resolve this conversation if #58 works for you.

Reply:
I went ahead and made #59 just to keep the work discrete.